dashboard/bin/delayed_job

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+
+# This script sets up and runs a Delayed Job worker in the background.
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
+require 'delayed/command'
+Delayed::Command.new(ARGV).daemonize
```

Comment:
Are we documenting how to run multiple workers somewhere? Is it still `bin/delayed_job -n 10`?

Reply:
generally, the documentation for running delayed job workers is here: https://github.com/collectiveidea/delayed_job#running-jobs

for us, the syntax would be `bin/delayed_job -n 10 restart`, with a caveat about lowering the number of jobs detailed here: https://github.com/code-dot-org/code-dot-org/pull/53976#discussion_r1339354141

additionally, I figured I would add the documentation about the caveat when we go to increase the number of workers, as illustrated here: 516132943947f391bc00550aeff0abaa14f0b109